### PR TITLE
added listKeyValue TLE for app config

### PR DESCRIPTION
**Pull request by bmoore-msft**
came up in a build session, not documented for templates yet but adding it here.  See:

https://docs.microsoft.com/en-us/rest/api/appconfiguration/configurationstores/listkeyvalue

### CODE REVIEW

File: assets/ExpressionMetadata.json [hunk 1]
```diff
@@ -458,6 +458,18 @@
                 }
             ]
         },
+        {
+            "name": "listKeyValue",
+            "expectedUsage": "listKeyValue(resourceName\/resourceIdentifier, apiVersion, KeyLabelObject)",
```
Comment (StephenWeatherford) [inline assets/ExpressionMetadata.json:463]: nit: 
```suggestion
            "expectedUsage": "listKeyValue(resourceName\/resourceIdentifier, apiVersion, keyLabelObject)",
```

File: assets/ExpressionMetadata.json [hunk 2]
```diff
@@ -458,6 +458,18 @@
...
+            "description": "Returns a value from the App Config store with Key/Label specified in the request. The KeyLabelObject parameters is a JSON object with the two properties \"key\" and \"label\" as required by the values in the store.",
```
Comment (StephenWeatherford) [inline assets/ExpressionMetadata.json:464]: How about an example?  Some of these are difficult to figure out.  For instance, I'm guessing keyLabelObject has to be a variable/param reference?  Is there a way to create a JSON object directly in TLE?

File: assets/ExpressionMetadata.json [hunk 3]
```diff
@@ -458,6 +458,18 @@
...
+            "minimumArguments": 3,
+            "maximumArguments": 3,
+            "returnValueMembers": [
+                {
+                    "name": "value"
+                }
+            ]
+        },
         {
```
Comment (StephenWeatherford) [inline assets/ExpressionMetadata.json:473]: We could also relatively easily add a field for a documentation link (https://docs.microsoft.com/en-us/rest/api/appconfiguration/configurationstores/listkeyvalue), if you think that makes sense.

File: assets/ExpressionMetadata.json (hunk 2 above)
Comment (bmoore-msft) [reply assets/ExpressionMetadata.json:464]: Can we provide json within the json?  Can certainly do that...

File: assets/ExpressionMetadata.json (hunk 3 above)
Comment (bmoore-msft) [reply assets/ExpressionMetadata.json:473]: It's not doc'd yet (separate bug for that) but we could link here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions#resource-functions

We don't actually doc each function in the template docs - could reference the REST API though that's a little out of context.  Could also reference quickstart but that link would eventually get broken

File: assets/ExpressionMetadata.json (hunk 2 above)
Comment (StephenWeatherford) [reply assets/ExpressionMetadata.json:464]: So, did you want to add something to the description here to clarify usage or leave it alone?

Comment (bmoore-msft) [reply assets/ExpressionMetadata.json]: I'm fine with it as is...

**Pull request merged**